src/components/UI/atoms/Divider/Divider.spec.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render, cleanup } from '<src>/helpers/testUtils';
+import Divider from './Divider';
+import 'jest-dom/extend-expect';
+
+afterEach(cleanup);
+
+describe('Divider', () => {
+  it('should render the Divider Component', () => {
+    const { container } = render(<Divider />);
+
+    expect(container.lastChild).toHaveStyle(`
+            border: 1px solid #B02091
+        `);
+  });
+
+  it('should render the Divider Component with width', () => {
+    const { container } = render(<Divider />);
+
+    expect(container.lastChild).toHaveStyle(`
+            width: md
```

Comment:
Huh what is this I thought the prop had changed 🤔 . Please test for height as well